Portal/src/Datahub.Portal/i18n/localization.fr.json

```diff
@@ -3073,5 +3073,84 @@
   "{0} credits consumed yesterday": "{0} crédits consommés hier",
   "{0} is already added to your workspace": "{0} est déjà ajouté à votre espace de travail",
   "{0} remaining credits of {1}": "{0} crédits restant de {1}",
-  "{0} successfully added to your workspace": "{0} ajouté avec succès à votre espace de travail"
+  "{0} successfully added to your workspace": "{0} ajouté avec succès à votre espace de travail",
+  "Shared Key Access is disabled. Please enable it in workspace settings to allow token generation.": "L'accès par clé partagée est désactivé. Veuillez l'activer dans les paramètres de l'espace de travail pour permettre la génération de jetons.",
+  "SAS Token generation is restricted. Please contact your workspace lead or administrators to get a token.": "La génération de jetons SAS est restreinte. Veuillez contacter le responsable de votre espace de travail ou les administrateurs pour obtenir un jeton.",
+  "Successfully updated workspace publishing settings.": "Mise à jour réussie des paramètres de publication de l'espace de travail.",
+  "Failed to update workspace publishing settings.": "Échec de la mise à jour des paramètres de publication de l'espace de travail.",
+  "(untitled dataset {0})": "(ensemble de données sans titre {0})",
+  "Select Dataset": "Sélectionner l'ensemble de données",
+  "TbsOpenGovPublishing.FileTypes.DataDictionary": "Dictionnaire de données",
+  "TbsOpenGovPublishing.FileTypes.Dataset": "Ressources de données",
+  "TbsOpenGovPublishing.FileTypes.Guide": "Guide",
+  "TbsOpenGovPublishing.FileTypes.ImsoApproval": "Approbation de l'IMSO",
+  "TbsOpenGovPublishing.FileTypes.Metadata": "Métadonnées",
+  "TbsOpenGovPublishing.FileTypes.SupportingDoc": "Documentation à l'appui",
+  "TbsOpenGovPublishing.Status.AwaitingApprovalCriteria.Description": "Veuillez remplir le formulaire des critères de publication d'un gouvernement ouvert.",
+  "TbsOpenGovPublishing.Status.AwaitingApprovalCriteria.Title": "Confirmer les critères de publication",
+  "TbsOpenGovPublishing.Status.AwaitingFiles.Description": "Veuillez sélectionner les fichiers à inclure dans la publication.",
+  "TbsOpenGovPublishing.Status.AwaitingFiles.Title": "Ajouter des fichiers",
+  "TbsOpenGovPublishing.Status.AwaitingImsoApproval.Description": "Veuillez obtenir l'approbation de l'IMSO et joindre le formulaire signé à la publication.",
+  "TbsOpenGovPublishing.Status.AwaitingImsoApproval.Title": "Confirmer l'approbation de l'IMSO",
```

Comment:
is there a translation for IMSO?

Reply:
I wasn't able to find one.

Reply:
Good find, I'll update those translations